rcl/test/rcl/test_subscription.cpp

```diff
@@ -179,6 +179,10 @@ TEST_F(CLASSNAME(TestSubscriptionFixture, RMW_IMPLEMENTATION), test_subscription
   //                probably using the count_subscriptions busy wait mechanism
   //                until then we will sleep for a short period of time
   std::this_thread::sleep_for(std::chrono::milliseconds(1000));
+#ifdef RMW_TIMESTAMPS_SUPPORTED
+  rcl_time_point_value_t pre_publish_time;
+  EXPECT_EQ(0, rcutils_system_time_now(&pre_publish_time)) << " could not get system time failed";  
```

Comment:
```suggestion
  EXPECT_EQ(RCUTILS_RET_OK, rcutils_system_time_now(&pre_publish_time))
    << " could not get system time failed";  
```

You might want to run `ament_uncrustify` over that, I don't know if I got the style just right.

Reply:
Took me two tries, but I think I got it now ,-)